dht.go

```diff
@@ -465,6 +465,10 @@ func (dht *IpfsDHT) peerFound(ctx context.Context, p peer.ID, queryPeer bool) {
 		if queryPeer {
 			dht.routingTable.UpdateLastSuccessfulOutboundQuery(p, time.Now())
 		}
+		// if we just added this peer, consider querying the DHT for more.
+		if added {
+			dht.fixRTIfNeeded()
```

Comment:
not sure if this is the right fix. Note: the `fixLowPeersRoutine` will actually end up calling back into this function. However, the signal channel has no buffer so we won't end up repeatedly raising the flag.

Reply:
@Stebalien Given that we already kick off refreshes when we discover a new connection or lose a peer from the RT, there's nothing new we'd accomplish by adding this here. However, this does make the code hard to reason about because of fix Routine and this guy calling each other. 

I wouldn't add it here.

Reply:
I agree.